package.json

```diff
@@ -21,13 +22,15 @@
   "dependencies": {
     "@tusbar/cache-control": "^0.3.1",
     "@types/esprima": "^4.0.2",
-    "lighthouse": "^8.6.0",
+    "lighthouse": "next",
```

Comment:
I worry that if there are breaking changes they will no longer be version tied, and older versions will fail to work. Does this seem like a reasonable concern?

Reply:
I'll change this to a specific nightly version. And modify again when 10.0 is actually released, ~Sept.

If you'd rather not cut an actual release to npm, that'd be fine. We can always tie Lighthouse's version of pubads to a branch or specific commit during the transition.